### PR TITLE
Feature/42994

### PR DESCRIPTION
**Pull request by Aekal**
dev-42994

### CODE REVIEW

File: Magento_Swatches/web/js/swatch-renderer.js [hunk 1]
```diff
@@ -1021,8 +1021,13 @@ define([
...
               //Category View
-              $this.parents('.product-item-info').find('.product-image-photo')
-                  .addClass($widget.options.classes.loader);
+              $this.parents('.product-item-info')
+                  .find('.lazyload-wrapper')
+                  .append(`
+                    <div class="loader loader--visible">
+                      <div class="loader__circle"></div>
+                    </div>
+                  `);
```
Comment (szafran89) [inline Magento_Swatches/web/js/swatch-renderer.js]: IMO you don't need literal string here, you can use regular quotes

File: Magento_Theme/requirejs-config.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
 var config = {
   map: {
     "*": {
+      "choices"            : "Magento_Theme/js/lib/choices",
       "closestPolyfill"    : "Magento_Theme/js/lib/closest-polyfill",
+      "lazysizes"          : "Magento_Theme/js/lib/lazysizes",
+      "lory"               : "Magento_Theme/js/lib/lory",
+      "noUiSlider"         : "Magento_Theme/js/lib/nouislider.min",
+      "picturefill"        : "Magento_Theme/js/lib/picturefill",
+      "svg4everybody"      : "Magento_Theme/js/lib/svg4everybody",
```
Comment (szafran89) [inline Magento_Theme/requirejs-config.js]: Why did you changed oreder of these items?  Should be added alphabetically

File: Magento_Swatches/web/js/swatch-renderer.js (hunk 1 above)
Comment (Aekal) [reply Magento_Swatches/web/js/swatch-renderer.js]: I need it for multiline strings, it's more readable

File: Magento_Theme/requirejs-config.js (hunk 2 above)
Comment (Aekal) [reply Magento_Theme/requirejs-config.js]: I grouped them by type then sorted alphabetically, libs are first then rest of scripts. If you don't like it I can switch to previous version

**Pull request merged**